int16 x
int16 y
int16 z
uint8 shoulderSnum
uint8 femurSnum
uint8 tibiaSnum